src/v2i-hub/CARMACloudPlugin/src/CARMACloudPlugin.cpp

```diff
@@ -604,13 +605,13 @@ void CARMACloudPlugin::ConvertString2Pair(std::pair<string,string> &str_pair, co
 QByteArray CARMACloudPlugin::UncompressBytes(const QByteArray compressedBytes) const
 {
     z_stream strm;
-	strm.zalloc = nullptr;//Refer to zlib docs (https://zlib.net/zlib_how.html)
-	strm.zfree = nullptr; 
-    strm.opaque = nullptr;
+	strm.zalloc = Z_NULL;//Refer to zlib docs (https://zlib.net/zlib_how.html)
```

Comment:
Whitespace in this function looks a little weird, but could just be github's rendering.